tools/Raven.Migrator/MigrationHelpers.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Globalization;
+using System.IO;
+using System.IO.Compression;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Raven.Migrator.MongoDB;
+
+namespace Raven.Migrator
+{
+    public static class MigrationHelpers
+    {
+        private const string StartObject = "{";
+        private const string EndObject = "}";
+        private const string RavenDocumentId = "@id";
+        private const string RavenCollection = "@collection";
+
+        public static void OutputClass(
+            AbstractMigrationConfiguration configuration,
+            object obj)
+        {
+            if (configuration.ConsoleExport)
+            {
+                var jsonString = JsonConvert.SerializeObject(obj);
```

Comment:
Maybe we can use EntityToBlittable here and use Sparrow to save memory (just performance enhancements)

Reply:
can't use Sparrow in the Migrator

Reply:
disable indentation?

Reply:
and I think that you can add Raven.Client and Sparrow.dll if needed.